incubator/kafka/notes.txt

```diff
@@ -0,0 +1,36 @@
+You can connect to Kafka by running a simple pod in the K8s cluster like this with a configuration like this:
+
+  apiVersion: v1
+  kind: Pod
+  metadata:
+    name: testclient
+    namespace: kafka
```

Comment:
I wouldnt set the namespace explicitly here. Users may not have configured this namespace for installation. 

You can make this a kubectl run command and avoid having the pod yaml here. Look at the postgresql chart as an example.

`kubectl run client -it --rm --image solsson/kafka:0.10.0.1 --command -- bash`